tools/crm_resource_runtime.c

```diff
@@ -2041,7 +2041,7 @@ cli_resource_execute(pe_resource_t *rsc, const char *requested_name,
     if (rsc->variant == pcmk_rsc_variant_group) {
         out->err(out, "Sorry, the %s option doesn't support group resources", rsc_action);
         return CRM_EX_UNIMPLEMENT_FEATURE;
-    } else if (rsc->variant == pe_container || pe_rsc_is_bundled(rsc)) {
+    } else if (pe_rsc_is_bundled(rsc)) {
```

Comment:
guess the behavioral change here is rather a fix or simply irrelevant

Reply:
it's irrelevant -- pe_rsc_is_bundled() will return true if rsc->variant is bundle

Reply:
well it will move to the upmost parent first and then check variant ... but probably still irrelevant ...

Reply:
Yep, bundles always have parent==NULL